homeassistant/components/hue/hue_sensor.py

```diff
@@ -0,0 +1,237 @@
+"""Support for the Philips Hue sensors as a platform."""
+import asyncio
+from datetime import timedelta
+import logging
+from time import monotonic
+
+import async_timeout
+
+from homeassistant.components import hue
+from homeassistant.helpers.event import async_track_point_in_utc_time
+from homeassistant.util.dt import utcnow
+
+
+DEPENDENCIES = ['hue']
+SCAN_INTERVAL = timedelta(seconds=5)
```

Comment:
Same

Reply:
This is referenced below. It's used to manage polling as `should_poll` is `False` for this latest suggested implementation.